tests/functional/spec/features/repeating_blocks/list_collector_repeating_blocks.spec.js

```diff
@@ -96,7 +96,7 @@ describe("List Collector Repeating Blocks", () => {
       await $(AnyOtherCompaniesOrBranchesPage.listEditLink(2)).click();
       await $(EditCompanyPage.companyOrBranchName()).setValue("Government");
       await click(EditCompanyPage.submit());
-      await expect(await $(AnyOtherCompaniesOrBranchesPage.listLabel(2)).getText()).toBe("Government");
+      await checkItemsInList(["ONS", "GOV", "Government"], AnyOtherCompaniesOrBranchesPage.listLabel);
```

Comment:
How come this was updated? A bit confusing now as I would have expected Gov to be have edited to be Government but it was MOD 😅

Reply:
Updated 👍